docs/reference/changelog.rst

```diff
@@ -4,13 +4,21 @@ Changelog
 See the `Releases page`_ on GitHub for a complete list of commits that are
 included in each version.
 
-2.2.0 (2025-Jan-16)
```

Comment:
This hasn't been released yet.  I'll date this once the PR is approved and ready to land.